### ISSUE TITLE
Dataset at www.eia.gov

### ISSUE THREAD

**Issue by nickrsan** [climate-mirror/datasets#86]
Name: 
Organization: US Energy Information Administration
Description URL: www.eia.gov
Download URL: 
File Types: Excel spreadsheets
Size: No Idea.  Huge 
Status:

**Comment by empirical-bayesian** [comment]
This has been replicated by The Azimuth Backup Project in its Issue #48.

**Comment by BethTrask** [comment]
I can't find these EIA data in the Azimuth Project.  Can you point me to them?   Specifically, these pages would be really important to save:
http://www.eia.gov/petroleum/data.cfm
http://www.eia.gov/naturalgas/data.cfm
https://www.eia.gov/consumption/data.php
I'm going to get started, downloading them chart by chart - but perhaps there are some far more technically savvy people who could do it faster and better than I can. (I'm non-technical, policy advocate at the environmental NGO, Environmental Defense Fund.)

**Comment by empirical-bayesian** [comment]
Beth,



I'll work on extracting these sections and putting them someplace you
can get to them this evening.


We are intending to make all our datasets public, but we want them all
copied to safe places other than just the mirrors in case someone gets
in and trashes the mirror servers.   You understand.


 - Jan





On Thu, Jan 26, 2017, at 16:58, BethTrask wrote:

> I can't find these EIA data in the Azimuth Project.  Can you point
> me to them?   Specifically, these pages would be really important
> to save:
> http://www.eia.gov/petroleum/data.cfm

> http://www.eia.gov/naturalgas/data.cfm

> https://www.eia.gov/consumption/data.php

>  I'm going to get started, downloading them chart by chart - but
>  perhaps there are some far more technically savvy people who could do
>  it faster and better than I can. (I'm non-technical, policy advocate
>  at the environmental NGO, Environmental Defense Fund.)
> — You are receiving this because you commented. Reply to this email
> directly, view it on GitHub[1], or mute the thread[2].
> 




Links:

  1. https://github.com/climate-mirror/datasets/issues/86#issuecomment-275527190

**Comment by empirical-bayesian** [comment]
Beth,

If I give them to you as gzip'd tarballs, is that okay?

 - Jan

**Comment by BethTrask** [comment]
Yes, absolutely!  That sounds wise. My team here also wants to make sure all these datasets are public, as well.  For example, assuming no one else is doing it, we are going to put up a site for the EPA Greenhouse Gas Reporting Program data.

Thanks, again.  It is really wonderful how everyone is coming together around this cause.

Beth
